extensions/amp-analytics/0.1/vendors/infonline_anonymous.json

```diff
@@ -0,0 +1,21 @@
+{
+  "vars": {
+    "ap": "1"
+  },
+  "transport": {
+    "beacon": false,
+    "xhrpost": false,
+    "image": true
+  },
+  "requests": {
+    "pageview": "${url}?ap=${ap}&dn=${dn}&cn=${cn}&st=${st}&co=${co}&cp=${cp}&au=${canonicalUrl}&ar=${documentReferrer}&ash=${screenHeight}&asw=${screenWidth}&tp=pageview",
```

Comment:
FYI: since we can't encode the `${url}` value, please make sure that your client specify it in the `requests` setting, not the `vars` setting. 
I think you've known this already : )